### PR TITLE
Dev/bruceppeng/1.0 double registry

### PR DESCRIPTION
**Pull request by pengkai242**
**Please provide issue(s) of this PR:**
Fixes # Added dual registration capability to NACOS Client V1.0

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Docs
- [ ] Discovery
- [ ] Route
- [ ] RateLimit
- [ ] CircuitBreaker
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: polaris-agent-adapter/pinpoint/pom.xml [hunk 1]
```diff
@@ -14,8 +14,9 @@
     <name>Polaris Agent Adapter Pinpoint</name>
     <modules>
         <!--<module>pinpoint-spring-cloud-2021-plugin</module>-->
-        <module>pinpoint-dubbo2-plugin</module>
-        <module>pinpoint-dubbox-plugin</module>
+<!--        <module>pinpoint-dubbo2-plugin</module>-->
```
Comment (chuntaojun) [inline polaris-agent-adapter/pinpoint/pom.xml:17]: 这里不要注释，自己调整下构件脚本

File: polaris-agent-build/bin/plugins [hunk 2]
```diff
@@ -1,2 +1 @@
-dubbo2
```
Comment (chuntaojun) [inline polaris-agent-build/bin/plugins:1]: 同上，不要注释掉，插件只是追加

File: polaris-pinpoint-agent-nacosv1-5/pinpoint-root.config [hunk 3]
```diff
@@ -0,0 +1,368 @@
+#
```
Comment (chuntaojun) [inline polaris-pinpoint-agent-nacosv1-5/pinpoint-root.config:1]: 为啥会有这个文件？

File: pom.xml [hunk 4]
```diff
@@ -14,7 +14,7 @@
     <url>https://github.com/polarismesh/polaris-java-agent</url>
 
     <properties>
-        <polaris.version>1.6.0-SNAPSHOT</polaris.version>
+        <polaris.version>1.6.0</polaris.version>
```
Comment (chuntaojun) [inline pom.xml:17]: 这里版本号先不要动

File: pom.xml [hunk 5]
```diff
@@ -37,6 +37,8 @@
         <alibaba.dubbo.version>2.6.12</alibaba.dubbo.version>
         <!-- apache dubbo (dubbo2) -->
         <apache.dubbo.version>2.7.15</apache.dubbo.version>
+        <!-- nacos client (1.4.1) -->
+        <nacos.client.version>1.4.1</nacos.client.version>
```
Comment (chuntaojun) [inline pom.xml:41]: nacos.v1.client.version

**Review by chuntaojun — APPROVED** [review]

File: polaris-agent-core/plugins/pom.xml [hunk 6]
```diff
@@ -16,9 +16,10 @@
     <packaging>pom</packaging>
 
     <modules>
-        <module>dubbo2</module>
-        <module>dubbox</module>
+<!--        <module>dubbo2</module>-->
```
Comment (andrewshan) [inline polaris-agent-core/plugins/pom.xml:19]: 这个应该不需要注释吧？

**Review by andrewshan — APPROVED** [review]
LGTM

**Pull request merged**